src/java.desktop/share/classes/javax/swing/border/AbstractBorder.java

```diff
@@ -85,6 +85,8 @@ public Insets getBorderInsets(Component c)       {
      * @param c the component for which this border insets value applies
      * @param insets the object to be reinitialized
      * @return the <code>insets</code> object
+     * @throws {@code NullPointerException} if {@code insets}
+     *         is {@code null}
```

Comment:
This does not compile, `@throws` expects a “plain” class name, monospaced font is applied automatically to `NullPointerException`.